nevergrad/optimization/optimizerlib.py

```diff
@@ -485,28 +487,25 @@ def _internal_tell(self, x: base.ArrayLike, value: float) -> None:
                 self.llambda = max(self.llambda, self.num_workers)
                 self.mu = self.llambda // 4
```

Comment:
@teytaud 
this is if if `self.num_workers > 1`
Can you explain why? I find it difficult to understand

Reply:
i have no idea... this "if" doesn't harm anything as far as I see, but it looks like it's useless.